### DISCUSSION TITLE
basic newbie question

### DISCUSSION THREAD

**Discussion by danbromberg** [Klocman/Bulk-Crap-Uninstaller#319] [General]
I just installed the portable version of BCUninstaller on a thumb drive since I didn't think 100+MB uninstaller would be worth it (why so large, I wondered?). But now that I poked around I like it.
Is there a difference between the portable version and the fully installed version on my PC?
Thanks,
Dan

**Comment by Klocman** [comment]
The installer only installs 32 or 64 bit version depending on your system, so the size is smaller. Other than that, BCU detects that it's installed and tweaks some things slightly, but overall it's going to be identical.

**Comment by 411621** [comment]
I'm running Windows 10 X64 and use interactive ESET as firewall.  I continuously get surprised of what pops up on once outbound and/or inbound access of apps or functions I never installed, never use or otherwise.  Despite attempts of disabling uninstalling etc. quite a few apps/services want a connection and continue with successive cumulative tries despite being blocked.  For example onedrivesetup, yourphone, Skype, msedgewebview etc
I'm relatively new to BCU and wonder is there a feature where I can strip down Windows of all the needless ballast (currently close to 300 processes are running after boot up)?
Thanks in advance!
Rob

**Reply by Klocman** [reply]
Check tools/startup manager. BCU won't automatically select what to remove, you have to select what you don't need manually.
